ecl/hql/hql.hpp

```diff
@@ -24,7 +24,6 @@
 #endif
 
 #include "hqlatoms.hpp"
-#include "build-config.h"
```

Comment:
These changes are also present in a separate PR, which should be merged first and this one rebased.